### ISSUE TITLE
Feature Request : add preview output

### ISSUE THREAD

**Issue by ArkhamInsanity** [lks-ai/ComfyUI-StableAudioSampler#9]
Simple as that, being able to listen to the output before choosing to save it or not (my storage would appreciate it :D ).
Thanks for this amazing work !

**Comment by lks-ai** [comment]
Yes, I will be adding perhaps a preview node for previewing the audio from within comfy.
I'm currently working on being able to output and accept the various audio nodes formats from other packs too.

**Comment by Amaru-Zeas** [comment]
wait how do you save the files? after running


![image](https://github.com/lks-ai/ComfyUI-StableAudioSampler/assets/131799050/651f35f5-08fd-45ab-aebe-cf02e47d28dd)
